atom/browser/api/atom_api_app.cc

```diff
@@ -183,7 +187,7 @@ void App::OnFinishLaunching() {
   auto browser_context = static_cast<AtomBrowserContext*>(
       AtomBrowserMainParts::Get()->browser_context());
   auto handle = Session::CreateFrom(isolate(), browser_context);
-  default_session_.Reset(isolate(), handle.ToV8());
+  g_wrap_session.Run(handle.ToV8());
```

Comment:
Yeah we can expose it as a builtin module, but we should keep it undocumented for now before we have full support for multi-session.